doc/api/inspector.md

```diff
@@ -0,0 +1,102 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+
+The `inspector.Session` class is a primary means of dispatching messages to
+the V8 inspector back-end and receiving message responses and notifications.
+
+### Constructor: new inspector.Session()
+<!-- YAML
+added: REPLACEME
+-->
+
+Create a new instance of the `inspector.Session` class. The resulted inspector
+session needs to be connected through [`session.connect()`][] before
+messages can be sent to the inspector backend.
+
+`inspector.Session` is is an [`EventEmitter`][] with the following events:
+
+### Event: 'inspectorNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when a notification from the V8 Inspector is received.
+
+### Event: INSPECTOR_PROTOCOL_METHOD
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when an inspector notification is recieved that has method field set
+to INSPECTOR_PROTOCOL_METHOD value.
+
+The following snippet installs a listener on the [`Debugger.paused`][]
+event, and prints the reason for program suspension whenever program
+execution is suspended (through breakpoints, for example):
+
+```js
+session.on('Debugger.paused', ({reason}) => console.log(reason));
+```
+
+### session.connect()
+<!-- YAML
+added: REPLACEME
+-->
+
+Connects a session to the inspector back-end. An exception will be thrown
+if there is already a connected session esteblished either through API or by
+a front-end connected to the Inspector WebSocket port.
+
+### session.post(method[, params][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* method {string}
+* params {Object}
+* callback {Function}
+
+Posts a message to the inspector back-end. Optional `callback` will be notified
+when a response is received. `callback` is a function that accepts two
+optional arguments - error and message-specific result.
+
+```js
+session.post('Runtime.evaluate', {'expression': '2 + 2'},
+             (error, {result}) => console.log(result.value));
+// Output: { type: 'number', value: 4, description: '4' }
+```
+
+Up-to-date version of the V8 inspector protocol is published on the
+[Chrome DevTools Protocol Viewer][].
+
+Node inspector supports all V8 domains.
```

Comment:
@addaleax all Inspector methods are namespaced to a domain, e.g. `Debugger.enable` and  `Profiler.start`.

A list of the currently supported domains is on the left of the protocol explorer at https://chromedevtools.github.io/debugger-protocol-viewer/v8/

"supports all V8 domains from the "Inspector Protocol" or "Chrome DevTools Protocol"" might make that clearer.